tools/create_github_release.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+"""
+Post-release script to download artifacts from azure pipelines and use them to create
+a GitHub release.
+
+Setup:
+ - Create an azure personal access token
+   - https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate
+   - You'll need read scope
+   - Save the token to somewhere like ~/.ssh/azurepat.txt
+ - Create a github personal access token
+   - https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token#creating-a-token
+   - You'll need repo scope
+   - Save the token to somewhere like ~/.ssh/githubpat.txt
+
+Run:
+
+python tools/create_github_release.py 1.7.0 ~/.ssh/azurepat.txt ~/.ssh/githubpat.txt
+"""
+
+import requests
+import sys
+import tempfile
+import urllib.parse
+from zipfile import ZipFile
+
+from azure.devops.connection import Connection
+from github import Github
+from msrest.authentication import BasicAuthentication
+
+def download_azure_artifacts(azure_access_token, tempdir):
+    """Download and unzip build artifacts from Azure pipelines.
+
+    :param str azure_access_token: string containing azure access token
+    :param str path: path to a temporary directory to save the files
+
+    """
+    # Create a connection to the azure org
+    organization_url = 'https://dev.azure.com/certbot'
+    credentials = BasicAuthentication('', azure_access_token)
+    connection = Connection(base_url=organization_url, creds=credentials)
+
+    # Find the build artifacts
+    build_client = connection.clients.get_build_client()
+    get_builds_response = build_client.get_builds('certbot', definitions='3')
+    build_id = get_builds_response.value[0].id
```

Comment:
This assumes that you want the artifacts from the most recent release. We could do it differently if we want to.

Reply:
In case GitHub displays this comment without context, this is a response to https://github.com/certbot/certbot/pull/8189/files#r465405156.

Thanks for flagging this, but I personally think that's fine and that it's a pretty safe assumption to make.

Reply:
It did in fact do that, thanks. Sounds good.